### DISCUSSION TITLE
guide-R/manipulation/pipe

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#24] [Forum]
# guide-R/manipulation/pipe

https://larmarange.github.io/guide-R/manipulation/pipe.html

**Comment by Valentin335** [comment]
Bonsoir ! Tout d'abord, merci pour le guide ; il est vraiment bien articulé. Cependant, je ne peux m'empêcher de vous faire une remarque bienveillante concernant la rédaction et l'orthographe :
« Il est fréquent d’enchainer » → Il est fréquent d’enchaîner
« car cela entraine » → car cela entraîne
« lorsque l’on effecture library(tidyverse) » → lorsque l’on effectue library(tidyverse)
« Voyons un example d’utilisation » → Voyons un exemple d’utilisation

Il y a aussi quelques erreurs concernant la ponctuation, mais ce n'est pas gênant. J'espère que vous ne prendrez pas mal cette remarque sur ces quelques erreurs mineures.

**Reply by larmarange** [reply]
Merci pour ces retours. Les corrections ont été effectuées : https://github.com/larmarange/guide-R/commit/6d2889c9b69d71ec7b15fde1fd4db119eac07fab 

Je reconnais qu'il y a pas mal d'erreurs typographiques et orthographiques par manque de temps pour la relecture des chapitres. (J'avoue privilégier la rédaction de nouveaux chapitres quand je peux me libérer un peu temps). Mais je suis preneur de tout retour comme le vôtre.

Bien cordialement
